src/queries/waterTemperature.js

```diff
@@ -0,0 +1,12 @@
+import { waterTemperatureEndpoint } from '../config';
+
+export const getWaterTemperature = async () => {
+  const fetchObj = {
+    method: 'GET',
+    headers: { Authorization: 'readonly:LjjhVGU46EBcKND9j3cX' }
+  };
+
+  return await fetch(waterTemperatureEndpoint, fetchObj)
+    .then((response) => response.json())
+    .catch((err) => console.error(err));
```

Comment:
isn't `useQuery` handling the error cases? for volunteer and consul we also have no `then...catch`

Reply:
Yeah, you're right, I'll update.